### PR TITLE
[SW-519] Introduce SW Models into Spark python pipelines

### PR DESCRIPTION
**Pull request by jakubhava**
ColumnPruner, H2OGBM and H2ODeepLearning ( H2OGBM backed by mojo, H2ODeepLearning backed by binary model ) can now be used in PySpark pipelines.

### CODE REVIEW

**Comment by jakubhava** [comment]
@mmalohlava when you have some free time please have a look.

I still need to make all available arguments to models in scala in python as well but after that it's final.

This makes mojo predictions available from pyspark

File: py/pysparkling/ml/algo.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    def setParams(self, ratio=1.0, featuresCols=[], predictionCol=None):
+        kwargs = self._input_kwargs
+        return self._set(**kwargs)
+
+    def _create_model(self, java_model):
+        return H2OGBMModel(java_model)
+
+
+class H2OGBMModel(JavaModel, JavaMLWritable, JavaMLReadable):
+    pass
```
Comment (mmalohlava) [inline py/pysparkling/ml/algo.py]: How do i create an instance of `H2OGBMModel` if i have MOJO in hand?

**Comment by mmalohlava** [comment]
+1 for merge

File: py/pysparkling/ml/algo.py (hunk 1 above)
Comment (jakubhava) [reply py/pysparkling/ml/algo.py]: This I still need to figure out. But I think that we just need to create a helper method on python side similar to ```def createFromMojo(is: InputStream, uid: String = Identifiable.randomUID("mojoModel")): H2OMOJOModel = {```

which would call this method using py4j and then use this as JavaModel

This is jira for it, marked for the release after this one https://0xdata.atlassian.net/browse/SW-611

**Comment by jakubhava** [comment]
https://0xdata.atlassian.net/browse/SW-612 jira for making all args available in python API, also marked for the next release

**Comment by jakubhava** [comment]
thx @mmalohlava for review!

File: py/pysparkling/ml/algo.py (hunk 1 above)
Comment (jakubhava) [reply py/pysparkling/ml/algo.py]: Already resolved in the latest code

**Pull request merged**